proxy/http/HttpSessionAccept.cc

```diff
@@ -29,6 +29,9 @@
 bool
 HttpSessionAccept::accept(NetVConnection *netvc, MIOBuffer *iobuf, IOBufferReader *reader)
 {
+  ink_assert(iobuf == nullptr);  // not set on this code path
+  ink_assert(reader == nullptr); // not set on this code path
```

Comment:
I thought these are set when this function is called from `ProtocolProbeTrampoline`. Is this dead code?
https://github.com/apache/trafficserver/blob/2af0dce59513edaefe4885bc5c77e48666246d08/proxy/ProtocolProbeSessionAccept.cc#L147

Reply:
I was clearly wrong about this. Time to refactor again.